tests/test_pg_exe.py

```
@@ -0,0 +1,135 @@
+    with patch.object(process, "_pg_bindir", return_value=None):
+        with pytest.raises(ExecutableMissingException, match=re.escape(config.exec)):
+            process._pg_exe(None, config)
+
+
+@pytest.mark.parametrize(
+    "error",
+    (
+        FileNotFoundError("pg_config"),
+        PermissionError("pg_config"),
+        subprocess.CalledProcessError(1, "pg_config"),
+    ),
```

Ruff (0.16.1)

[warning] Wrong values type in `pytest.mark.parametrize` expected `list`

Use `list` for parameter values

(PT007)